test/base.jl

```diff
@@ -50,6 +50,12 @@ end
   @test fmap(x -> x + 10, bf) == Base.Broadcast.BroadcastFunction(Bar(13.3))
 end
 
+@testset "Returns" begin
+  ret = Returns([0, pi, 2pi])
+  @test Functors.functor(ret)[1] == (value = [0, pi, 2pi],)
+  @test Functors.functor(ret)[2]((value = 1:3,)) === Returns(1:3)
+end
```

Comment:
Forgot this, I think it needs a VERSION guard too. If that's enough to pass, merge when ready.

Reply:
```suggestion
VERSION >= v"1.7" && @testset "Returns" begin
  ret = Returns([0, pi, 2pi])
  @test Functors.functor(ret)[1] == (value = [0, pi, 2pi],)
  @test Functors.functor(ret)[2]((value = 1:3,)) === Returns(1:3)
end
```